docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -391,6 +451,88 @@ def my_io_manager(_):
     return MyIOManager()
 ```
 
+### Per-input IO Manager
```

Comment:
similarly unsure where this section should be (same concerns as the per input io management for assets)